gooddata-sdk/gooddata_sdk/catalog/permission/declarative_model/dashboard_assignees.py

```diff
@@ -0,0 +1,40 @@
+# (C) 2023 GoodData Corporation
```

Comment:
Please, add imports of all new classes here: 

https://github.com/gooddata/gooddata-python-sdk/blob/master/gooddata-sdk/gooddata_sdk/__init__.py

Then we can import them directly e.g.

`from gooddata_sdk import CatalogUserAssignee`

After you add imports to that file. Please adapt changes in `test_catalog_permission.py` and import classes only `from gooddata_sdk import ...`

Reply:
done